ynput/OpenPype
# 3dsmax: refactor load plugins

## Changelog Description
### Refactor 3dsmax Load Plugins
This pull request addresses part of the GitHub issue https://github.com/ynput/OpenPype/issues/4982 by refactoring any load plugins using `rt.execute()`. The primary focus of the refactoring is to replace the usage of `rt.execute()` with the appropriate implementation using pymxs.

This refactoring improves the overall code quality, performance, and safety of the 3dsmax publish plugins, bringing them in line with best practices and leveraging the full capabilities of the pymxs API.

The reasons for this refactoring are outlined in more detail in the GitHub issue https://github.com/ynput/OpenPype/issues/4982.

## Testing notes:
1. check that the `load_model_fbx` **update** function is working as intended 
2. check that the `load_pointcache` **load** function is working as intended 
3. check that the `load_model` **load** function is working as intended
